### DISCUSSION TITLE
Wrapped header

### DISCUSSION THREAD

**Discussion by mlapenna7** [KevinVandy/material-react-table#726] [Q&A]
I am trying to find a way to have a multi row header above a column.  Some my data is vary narrow (like a single digit or yes/no) but the column header might be two words (like ARM COUNT).  This cuts off the header and shows 3 dots.  Granted the full column title is available in the tooltip.  But it would be cool if I could insert a \n into the header string to force multi-row.  Is there another way to do it?  Apologies but I am still pretty new to material-react-table.

**Comment by KevinVandy** [comment]
Insert a `<br />` in a custom Header

**Reply by mlapenna7** [reply]
Thank you - not sure why I didn't think to try that.  By the way, I took the plunge and moved away from material-table to your MRT.  So glad I did - you've done amazing work.
